src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -254,8 +254,10 @@ public void testChoiceParameterSnippetizer() throws Exception {
         r.waitUntilNoActivity(); // #1 built automatically
         assertEquals(1, p.getBuilds().size());
         r.assertBuildStatusSuccess(p.scheduleBuild2(0)); // #2
+        Thread.sleep(500L); // WorkflowRun performs log rotation asynchronously.
```

Comment:
This test was failing consistently on my machine after the updates. I did not see a trivial way to wait for [this task](https://github.com/jenkinsci/workflow-job-plugin/blob/7a9d13220aacd4b535cc70976f8af082e7820644/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java#L618-L624) to complete before continuing and so just added a  call to `Thread.sleep`...